src/main/java/uk/gov/companieshouse/api/strikeoffobjections/controller/ObjectionController.java

```diff
@@ -439,4 +442,49 @@ public ResponseEntity deleteAttachment(
             );
         }
     }
+
+    @GetMapping("/{objectionId}/attachments/{attachmentId}/download")
+    public ResponseEntity<Void> downloadAttachment(@PathVariable String companyNumber,
+                                                   @PathVariable String objectionId,
+                                                   @PathVariable String attachmentId,
+                                                   @RequestHeader(value = ERIC_REQUEST_ID_HEADER) String requestId,
+                                                   HttpServletResponse response) {
+        Map<String, Object> logMap = new HashMap<>();
+        logMap.put(LOG_COMPANY_NUMBER_KEY, companyNumber);
+        logMap.put(LOG_OBJECTION_ID_KEY, objectionId);
+
+        try{
+            apiLogger.infoContext(
+                    requestId,
+                    "GET /{objectionId}/attachments/{attachmentId}/download request received",
+                    logMap
+            );
+            FileTransferApiClientResponse downloadServiceResult = objectionService.downloadAttachment(
+                    requestId, objectionId, attachmentId, response);
+            return ResponseEntity.status(downloadServiceResult.getHttpStatus()).build();
+        } catch (HttpClientErrorException | HttpServerErrorException e) {
+            apiLogger.errorContext(
```

Comment:
In PTF I was getting issues with re-using the logMap as the logger would modify the map and place the message into it. On subsequent calls it would see the map already had a message from the first call and not replace it, resulting in the old message being displayed. If this is logging correctly I don't understand how it is, I think we should use a new map for each log call. You can see how it works here https://github.com/companieshouse/structured-logging/blob/master/src/main/java/uk/gov/companieshouse/logging/StructuredLogger.java

Reply:
This looks like something we need to be changing more widely there is a ticket to do this on the backlog OBJ-141 - i'll add this to the ticketas it provides some guidence as to what we need to be aiming for.

Reply:
That story is to centralise the logging. I'll create a new one to ensure it does not get ignored.

Reply:
Ok we should link it to the 141 issue thought